src/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+function App() {
+  return <h1>Welcome to Barefoot Nomad</h1>;
+}
```

Comment:
it could be much better if this App function was imported from a separate file which is App.js instead of being a function in index.js. Having them separate, will help us to hold our other components in App.js.

Reply:
that's great idea, let me create the App component